packages/apps/dev-wallet/src/modules/db/db.service.tsx

```diff
@@ -59,13 +27,12 @@ export const setupDatabase = execInSequence(async (): Promise<IDBDatabase> => {
   const result = await connect(DB_NAME, DB_VERSION);
   let db = result.db;
   if (result.needsUpgrade) {
+    broadcast('migration-started');
     const oldVersion = result.oldVersion;
     if (oldVersion === 0) {
       console.log('creating new database');
       createTables(db);
-      return db;
-    }
-    if (oldVersion < 37) {
+    } else if (oldVersion < 37) {
       const confirmed = confirm(
         'You’re using an outdated database version that doesn’t support migration. To continue using the app, all data must be wiped. Do you want to proceed?',
       );
```

Comment:
I saw the [conf](https://github.com/kadena-community/kadena.js/pull/2709/files#diff-d4909648622d751e781bd4ca43cabe1e48da9213bffe170509c3c8a3b4a0b3d5) has a `DB_WIPE_ON_VERSION_CHANGE: true`, is this working together?

Reply:
no the DB_WIPE_ON_VERSION_CHANGE option is obsolete now, I'll remove it